Charts/qtest-insights/templates/deployment.yaml

```diff
@@ -125,6 +125,16 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.secrets.sessionsName }}
                   key: client.jdbc.postgres.password
+            - name: INSIGHTS_RO_DB_PW
```

Comment:
same here. It is not used anywhere. It probably had to be deleted. But as it is not used anywhere, I think it's ok to release it.